package.json

```diff
@@ -31,7 +31,7 @@
   },
   "homepage": "https://github.com/AndyOGo/stylelint-declaration-strict-value#readme",
   "peerDependencies": {
-    "stylelint": "^7.0.0 || ^8.0.0 || ^9.0.0"
+    "stylelint": "^7.0.0 || ^8.0.0 || ^9.0.0" || "^10.0.0"
```

Comment:
This looks like a typo (additional quote) and won't parse correctly as JSON. It should be:
`"stylelint": "^7.0.0 || ^8.0.0 || ^9.0.0" || ^10.0.0"`

Reply:
Thanks @Cherry